foundryup-zksync/foundryup-zksync

```diff
@@ -250,7 +301,7 @@ OPTIONS:
     -p, --path      Build and install a local repository
     -j, --jobs      Number of CPUs to use for building Foundry (default: all CPUs)
     --arch          Install a specific architecture (supports amd64 and arm64)
-    --platform      Install a specific platform (supports win32, linux, and darwin)
```

Comment:
are we officially dropping support for win32? I think we should document that properly then